File: docs-site/android/pubsub.kt
client.on("chat.room1") { payload -> 
  Log.d("Message", payload)
}

client.on("CONNECTED") {
  Log.d("SDKEvent", "Connected to Relay!")

  val message = buildMap<String, Any> {
      put("user_name", "John Doe")
      put("message", "How's it going fam?")
  }

  val sent = client.publish("chat.room1", message)
  Log.d("Message", "Message sent => $sent")
}
                      
client.connect()

File: docs-site/python/pubsub.py
def callback_fn(data):
    print(data)

def on_connected():
  print("Connected to Relay!")

  sent = await client.publish("chat.room1", {
      "user_name": "John Doe",
      "message": "How's it going fam?"
  })

  print(f"Message sent => {sent}")

await client.on("chat.room1", callback_fn)
await client.on(Realtime.CONNECTED, on_connected)

client.connect()